website/docs/cloud-docs/policy-enforcement/define-policies/prewritten-sentinel.mdx

```diff
@@ -0,0 +1,129 @@
+
+1. Browse the policy libraries available in the [Terraform registry](https://registry.terraform.io/browse/policies).
+1. Click on a policy library and click **Choose policies**.
+Select the policies you want to implement. The registry generates code in the **USAGE INSTRUCTIONS** box.
+1. Click **Copy Code Snippet** to copy the code to your clipboard. 
+1. Create a GitHub repository to store the policies and the policy set configuration file.
+1. Create a file called `sentinel.hcl` in the repository. 
+1. Paste the code from your clipboard into `sentinel.hcl` and commit your changes. 
+1. Complete the instructions for [connecting the policies to your workspace](#connect-policies-to-your-workspace).
+
+### Create a fork of the policy libraries 
+
+Create a fork of the repository containing the policies you want to implement. Refer to the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) for instructions on how to create a fork. 
+
+HashiCorp Sentinel policy libraries include a `sentinel.hcl` file. You can modify the file to customize your policy set. Refer to [Sentinel Policy Set VCS Repositories](/terraform/cloud-docs/policy-enforcement/manage-policy-sets/sentinel-vcs) for additional information.
+
+After forking the repository, complete the instructions for [connecting the policies to your workspace](#connect-policies-to-your-workspace).
+
+
+### Add the Terraform module to your configuration 
+This method enables you to connect the policies to workspaces in the Terraform configuration file. As a result, you can skip the instructions described in [Connect policies to your workspaces](#connect-policies-to-your-workspaces). 
+	 
+1. Go to the [module in the Terraform registry](https://registry.terraform.io/modules/hashicorp-modules/aws-cis/sentinel/latest) and copy the code generated in the **Provision Instructions** tile.  
+1. Add the `module` block to your Terraform configuration and define the following arguments: 
+   - `source`: Specify the path to the module you downloaded.
+   - `tfe_organization`: Specify the name of your organization on Terraform Enterprise or HCP Terraform.
+   - `policy_set_workspace_names`: Specify a list of workspace names that you want to apply the policies to. 
+
+   The following example configuration applies invokes the module for `target_workspace_1`:
+
+   ```hcl
+   module "cis_v1-2-0_policies" {
+      source = "../prewritten-policy"
+      name                             = "cis-1-2-0"
+      tfe_organization                 = "<your-tfe-org>"
+      policy_set_workspace_names       = ["target_workspace_1"]
+   }
+   ```
+
+1. Run `terraform plan` to view the plan.
+1. Run `terraform apply` to apply the changes.
+
+After successfully running the `terraform apply` command, Terraform evaluates Sentinel policies for each run of the workspaces you specified.
```

Comment:
Are we referring to the apply command the user just ran, or apply commands in the workspaces they just added the policies to?